server/v2/routes.js

```diff
@@ -0,0 +1,36 @@
+const express = require('express');
+const graphqlHTTP = require('express-graphql');
+const schema = require('./schema');
+const rootValue = require('./queries');
+
+const router = express.Router();
+
+router.post('/collect', (req, res) => {
+  req.app.locals.db.collection('weather/v2')
+    .insertOne({
+      temperature: req.body.data.temperature,
+      pressure: req.body.data.pressure,
+      humidity: req.body.data.humidity,
+      windSpeed: req.body.data.windSpeed,
+      windDirection: req.body.data.windDirection,
+      rain: req.body.data.rain,
+      createdAt: new Date()
+    }, (err, doc) => {
+      if (err) {
+        res.status(500).json({
+          error: 'Failed to create new weather data point.'
+        });
+      } else {
+        req.app.locals.wss.send('v2', doc.ops[0]);
+        res.status(201).json(doc.ops[0]);
+      }
+    });
+});
+
+router.use('/graphql', graphqlHTTP({
+  graphiql: process.env.NODE_ENV !== 'production',
```

Comment:
I know this is a best practice. But for the PrecipiStation, would it be neat to have that in prod too? 🤔 I could see myself dorking around with queries on my phone and then eventually turning those into UI / widgets / something

Reply:
That's actually a good idea because I also want to mess with production db stuff from time to time. These are read only endpoints anyway, so I don't see much harm in allowing public access.